tests/testthat/test-copy-model-from.R

```diff
@@ -213,3 +213,53 @@ test_that("copy_model_from(.new_model=NULL) errors when no models are valid inte
     new_mod2 <- copy_model_from(new_mod1)
   }, regexp = "no models.+integer")
 })
+
+test_that("copy_model_from works with various PROBLEM declarations [BBR-CMF-007]", {
```

Comment:
Thanks for the extensive tests.

(I know it's just copied from above, but might as well avoid adding new test IDs at this point.)